client/stacks/lib/views/stacks/editors/basestackeditorview.coffee

```diff
@@ -31,6 +31,7 @@ module.exports = class BaseStackEditorView extends IDEEditorPane
     options.contentType = contentType
     options.file        = FSHelper.createFileInstance
       path: "localfile:/stack-#{Date.now()}.#{contentType}"
+      dismissSaveDialogModal: yes
```

Comment:
this option is not related with the `File` instance here, instead you can `off` `'ace.requests.save'` event on ace instance.